edxval/settings.py

```diff
@@ -187,3 +187,17 @@
     CIELO24_BASE_API_URL="https://sandbox.cielo24.com/api",
     CIELO24_LOGIN_URL="https://sandbox.cielo24.com/api/account/login"
 )
+
+# Required by Django 2.2 to run management commands.
+TEMPLATES = [
```

Comment:
In the future, I'd get changes like this into a separate PR ahead of the squash migrations, so the squash migrations will contain nothing but the squash.